tests/browser/mobile-compatibility.spec.ts

```
@@ -0,0 +1,127 @@
+      const pageObj = await context.newPage()
+
+      // Navigate to the test page
+      await pageObj.goto(page.url)
+
+      // Wait for page to fully load
+      await pageObj.waitForLoadState('networkidle')
+
+      // Ensure directory exists for screenshots
+      await pageObj.evaluate(() => {
+        try {
+          require('fs').mkdirSync('./test-results/mobile', { recursive: true })
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)